docker-compose.yaml

```
@@ -0,0 +1,13 @@
+version: "3.8"
+services:
+
+  db:
+    volumes: 
+    - testvolume:/home/scribe-server/volume
```

```suggestion
    - testvolume:/var/lib/mysql
```

My understanding is that we likely want to store within a persistent volume what is actually inside `/var/lib/mysql` - as that is where the data gets stored for a MariaDB. For context, this can be useful so that any test data is still kept around even when the container is stopped.